src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ExceptionTest/ExceptionTest.cs

```diff
@@ -201,11 +201,21 @@ public static void ExceptionTests()
         }
 
         // Synapse: 110003;Invalid user or password
-        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
```

Comment:
Are we including Azure endpoint to this test? Why was is excluded and why are we including them now? Basically azure endpoints should not be included in this test and failures should not happen here.

Reply:
It was excluded in David's PR #2646. Before that PR was approved this unit test has been passing for Azure jobs until the introduction of Managed Identity Authentication.